### DISCUSSION TITLE
Collapsed Elements being shown by Linking to Anchor

### DISCUSSION THREAD

**Discussion by josh-at-fieldsresearch** [twbs/bootstrap#32497] [General]
Would it be possible for a 'top level; collapsed element to expand automatically if the anchor for the section is reached?

For example, imagine we have a page with the following collapsed sections
#SavingsRates
#LoanRates
#CheckingRates

If the page is reached via {URL}#LoanRates, the section would expand.   Likewise, if a table of contents on the page links to the section, it would expand.

**Comment by patrickhlauke** [comment]
that sounds more like something that a tiny separate plugin/extension to bootstrap should handle, rather than being a native part of bootstrap itself. conceptually, on load, you want to check if there's a fragment identifier in the URL, and if so trigger the respective control to open it
